force-app/main/default/classes/OpportunityCallout.cls

```diff
@@ -0,0 +1,38 @@
+public class OpportunityCallout {
+	@future(callout=true)
+	public static void postInfo(
+		String endpoint,
+		String partnerName,
+		String opportunityName,
+		Decimal amount,
+		String type
+	) {
+		Http http = new Http();
+		HttpRequest request = new HttpRequest();
+		request.setEndpoint(endpoint);
+		request.setMethod('POST');
+		request.setHeader('Content-Type', 'application/json;charset=UTF-8');
+		request.setBody(
+			'{"Name":"' +
```

Comment:
ok for now, but better to use additional class for request, like
public class OpportunityInfoRequest{
public String Name;
public String PartnerName;
etc..
}

and then prepare body using JSON.serialize